src/main/java/uk/ac/cam/cl/dtg/segue/configuration/SegueGuiceConfigurationModule.java

```diff
@@ -854,13 +854,25 @@ private static StatisticsManager getStatsManager(final UserAccountManager userMa
     @Inject
     private static SegueJobService getSegueJobService(final PostgresSqlDb database) {
         if (null == segueJobService) {
-            SegueScheduledJob PIISQLJob = new SegueScheduledDatabaseScriptJob("PIIDeleteScheduledJob", "SQLMaintenance",
-                    "SQL scheduled job that deletes PII", "0 0 2 * * ?", "db_scripts/scheduled/pii-delete-task.sql");
-
-            SegueScheduledJob cleanUpOldAnonymousUsers = new SegueScheduledDatabaseScriptJob("cleanAnonymousUsers", "SQLMaintenance",
-                    "SQL scheduled job that deletes old AnonymousUsers", "0 30 2 * * ?", "db_scripts/scheduled/anonymous-user-clean-up.sql");
-
-            segueJobService = new SegueJobService(Arrays.asList(PIISQLJob, cleanUpOldAnonymousUsers));
+            SegueScheduledJob PIISQLJob = new SegueScheduledDatabaseScriptJob(
+                    "PIIDeleteScheduledJob",
+                    "SQLMaintenance",
+                    "SQL scheduled job that deletes PII",
+                    "0 0 2 * * ?", "db_scripts/scheduled/pii-delete-task.sql");
+
+            SegueScheduledJob cleanUpOldAnonymousUsers = new SegueScheduledDatabaseScriptJob(
+                    "cleanAnonymousUsers",
+                    "SQLMaintenance",
+                    "SQL scheduled job that deletes old AnonymousUsers",
+                    "0 30 2 * * ?", "db_scripts/scheduled/anonymous-user-clean-up.sql");
+
+            SegueScheduledJob cleanUpExpiredReservations = new SegueScheduledDatabaseScriptJob(
+                    "cleanUpExpiredReservations",
+                    "SQLMaintenence",
+                    "SQL scheduled job that deletes expired reservations for the event booking system",
+                    "0 0 2 * * ?", "db_scripts/scheduled/expired-reservations-clean-up.sql");
```

Comment:
This looks fine. I did notice we have been offsetting the jobs start time. I am not sure why unless we have a limited number of db connections in the pool and we are trying to just use one at a time for quartz things.

Either way could be worth considering shifting the start time by a few minutes - will leave it up to you though.

Reply:
Yes, I was going to leave the final decision on the best start time to @jsharkey13 as he's more likely to know when the DB is not being overloaded at night…